packages/shared/tsconfig.eslint.json

```diff
@@ -10,6 +10,7 @@
     "**/.eslintrc.js",
     "__tests__/**/*.ts",
     "__tests__/**/*.tsx",
-    "__mocks__/**/*.ts"
+    "__mocks__/**/*.ts",
+    "__mocks__/**/*.tsx"
```

Comment:
I've seen this in shared and extension, don't they share the shared ones?

Reply:
From what I can see no, they extend the root one, and even if they did tsconfig is not deeply extendable it will always overwrite top level properties of extended config.